src/compiler/xquery/serialize/disable-escaping.xsl

```diff
@@ -23,7 +23,7 @@
       <xsl:param name="input" as="xs:string?" />
 
       <xsl:sequence select="
-         doc('')
+         doc(static-base-uri())
```

Comment:
Some bats tests that use XProc failed under Saxon 11 without a change here.

Reply:
Reference for the XML Calabash change in behavior: https://github.com/ndw/xmlcalabash1/issues/343

Reply:
Resolved with 093f2bcffa58178f36efac68d2771df75bbc1a90. We don't need this compiler change after all, if we use XML Calabash 1.5.6-110 with Saxon 11.